apps/condo/domains/common/utils/serverSchema/index.js

```diff
@@ -40,26 +58,22 @@ class GqlWithKnexLoadList {
         let skip = 0
         let newchunk = []
         let all = []
-        let haveNotifiedAboutTooManyObjs = false
+        let tooManyReturnedMemory = []
```

Comment:
It seems confusing to me

Maybe: tooManyReturnedLogsSentCounter? And why can't we just store number, like

1. If we have logged only 1 time, this var will be 1
2. If we have logged 2 times, this var will be 2 

etc